website/docs/api/transformer.md

```diff
@@ -453,7 +453,7 @@ using the `@spacy.registry.span_getters` decorator.
 > #### Example
 >
 > ```python
-> @spacy.registry.span_getters("sent_spans.v1")
+> @spacy.registry.span_getters("custom_sent_spans")
 > def configure_get_sent_spans() -> Callable:
 >     def get_sent_spans(docs: Iterable[Doc]) -> List[List[Span]]:
 >         return [list(doc.sents) for doc in docs]
```

Comment:
Come to think about it, this example is a little confusing? The code is the actual implementation for `sent_spans`  in `spacy-transformers`, so there would be no need to register a similar "custom" function in the "spacy" registry? I think we should either use an actual custom example that is not built-in, or show the code as it is in spacy-transformers.

Reply:
Hmmm yeah, I see your point... the main reason I renamed this is because if we keep the naming consistent, it would have to be named `spacy-transformers.sent_spans.v1`, and if a user just copy-pasted this example, they'd immediately see an error because (obviously) that function already exists 🤔

Reply:
Meh, let's just keep it as is then :-)